<launch>
    <node pkg="laser_scanner_infoscreen" name="tracking_client" type="track_objects_client" launch-prefix="valgrind"  output="screen"/>
    <node pkg="laser_scanner_infoscreen" name="tracking_server" type="track_objects_server" output="screen"/>
    <!-- <node pkg="laser_scanner_infoscreen" name="measure_biometrics" type="turtlesim_node"/> -->

</launch>
